### DISCUSSION TITLE
Can I show multiple text Column badge()

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9756] [General]
## Description

In my page I use TextColumn::make()->badge() in this I want to show two badge like the attendance status show two badge data like, present , overtimes how to make this?

**Comment by ShaungBhone** [comment]
Is your data json?

**Comment by ayemin-tun** [comment]
Yes

**Reply by ShaungBhone** [reply]
https://filamentphp.com/docs/3.x/tables/columns/text#using-a-list-separator
